examples/counter/src/Main.purs

```diff
@@ -41,8 +41,8 @@ ui = H.component { render, eval, initialState }
 main :: Eff (H.HalogenEffects ()) Unit
 main = runHalogenAff do
   body <- awaitBody
-  driver <- runUI ui body
-  setInterval 1000 $ driver (H.action Tick)
+  app <- runUI ui body
+  setInterval 1000 $ app.query (H.action Tick)
```

Comment:
Ah ok, I did check that! https://gist.github.com/garyb/e6aacc4019a1781d846a2a82e203c41b